scripts/smeshr/environment.yaml

```diff
@@ -0,0 +1,20 @@
+name: smeshr
+channels:
+- conda-forge
+dependencies:
+- cdo=2.2
```

Comment:
I think we discovered that the latest CDO version does also work. Should we pin to that?

Reply:
Hmm, I remember a discussion about some of the initial testing you had with CDO 2.4 but had thought the recommendation was to stay with 2.2.0? I've got some notes from our last call about the UU group members checking CDO versions for 2.0.5 and 2.2.0, but not 2.4.0.

Reply:
It's pretty minor and logged an issue #67